### ISSUE TITLE
Code mirror keybindings interfere with some keyboard layouts

### ISSUE THREAD

**Issue by amir-zeldes** [korpling/ANNIS#555]
By default, CodeMirror uses emacs-y keyboard bindings, which some languages use for input. For example, on the Mac Coptic keyboard layout, option+f inputs the letter phi (the same is true for option+d). This does not work in the AQL box due to keybindings from codemirror. 

To reproduce:
1. follow instructions to install Mac Coptic layout here: http://www.evertype.com/fonts/coptic/
2. activate the layout in a word processor and type option+f - works fine
3. do the same in AQL box - no character is printed

The following issue suggests a fix, which I can confirm works in another CodeMirror project I have:

https://github.com/codemirror/CodeMirror/issues/3254

Basically on initializing the CodeMirror textarea, the bindings should be deleted:

```
delete CodeMirror.keyMap.emacsy["Alt-F"]; 
delete CodeMirror.keyMap.emacsy["Alt-D"];
```

**Comment by thomaskrause** [comment]
From what I see in the documentation these keybindings should be only active if the mode is explicitly set to "emacs" and if we have the emacs keybinding file included. Both should not be the case. I think the functionality of both  is not really needed. I have to check on a mac why these key bindings are triggered (the solution would be to not support emacs keybindings at all)
